From a0f6e692504ec5f3edd8dee52b39d963395084ab Mon Sep 17 00:00:00 2001
Message-Id: <a0f6e692504ec5f3edd8dee52b39d963395084ab.1684276665.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1684276664.git.sharlatanus@gmail.com>
References: <cover.1684276664.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Tue, 16 May 2023 21:21:35 +0100
Subject: [PATCH 12/19] gnu: python-astroquery: Use Gexp.

* gnu/packages/astronomy.scm (python-astroquery): Use Gexp to simplify
package style.
---
 gnu/packages/astronomy.scm | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index be4961b49e..5f5de193d0 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -1280,15 +1280,15 @@ (define-public python-astroquery
         (base32 "1vhkzsqlgn3ji5by2rdf2gwklhbyzvpzb1iglalhqjkkrdaaaz1h"))))
     (build-system python-build-system)
     (arguments
-     `(#:phases
-       (modify-phases %standard-phases
+     (list
+      #:phases
+       #~(modify-phases %standard-phases
          (add-before 'check 'writable-home
            (lambda _                    ; some tests need a writable home
              (setenv "HOME" (getcwd))))
          (replace 'check
-           (lambda* (#:key inputs outputs tests? #:allow-other-keys)
+           (lambda* (#:key tests? #:allow-other-keys)
              (when tests?
-               (add-installed-pythonpath inputs outputs)
                (invoke "python" "-m" "pytest" "--pyargs" "astroquery"
                        ;; Skip tests that require online data.
                        "-m" "not remote_data")))))))
-- 
2.39.2

